pkgs/tools/admin/pulumi-packages/pulumi-language-python.nix

```diff
@@ -8,25 +8,16 @@ buildGoModule rec {
 
```

Comment:
Inheriting  `sdkVendorHash` (in line 7) is no longer necessary

Reply:
aha, removed it